flocker/control/_model.py

```diff
@@ -617,9 +628,15 @@ class Deployment(PClass):
     :ivar PSet nodes: A set containing ``Node`` instances
         describing the configuration of each cooperating node.
     :ivar Leases leases: A map of ``Lease`` instances by dataset id.
+    :ivar PersistentState persistent_state: The non-discoverable persistent
+        state of the cluster. (Note: XXX This should idealy be a sibling to the
+        configuration of the cluster, instead of child; but the required
+        refactoring doesn't seem worth it currently, and can be done later if
+        ever).
```

Comment:
Putting this here rather than as a sibling in `Configuration` or in a separate file isn't ideal, but required less changes (i.e. none) to `ConfigPersistenceService` to handle. Making it a sibling would have required adding new `.get` and `.save` methods (and probably renaming the existing ones for consistency). Adding a separate file would have in addition required refactoring the upgrade logic to dealt with two files (and there is the question of whether we might at some point need cross-file upgrades).

Putting it here is a bit of expediency, but I think the only cost is that it is slightly confusing (that is, I don't think it will cause us any development headaches).